boards/arm/ip_k66f/pinmux.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020 DENX Software Engineering GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <fsl_port.h>
+
+static int ip_k66f_pinmux_init(struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+#ifdef CONFIG_PINMUX_MCUX_PORTA
+	struct device *porta =
+		device_get_binding(CONFIG_PINMUX_MCUX_PORTA_NAME);
+#endif
+#ifdef CONFIG_PINMUX_MCUX_PORTB
+	struct device *portb =
+		device_get_binding(CONFIG_PINMUX_MCUX_PORTB_NAME);
+#endif
+#ifdef CONFIG_PINMUX_MCUX_PORTC
+	struct device *portc =
+		device_get_binding(CONFIG_PINMUX_MCUX_PORTC_NAME);
+#endif
+#ifdef CONFIG_PINMUX_MCUX_PORTD
+	struct device *portd =
+		device_get_binding(CONFIG_PINMUX_MCUX_PORTD_NAME);
+#endif
+#ifdef CONFIG_PINMUX_MCUX_PORTE
+	struct device *porte =
+		device_get_binding(CONFIG_PINMUX_MCUX_PORTE_NAME);
+#endif
+
```

Comment:
We are defining various port[a-e] variables above, but only porta is used below. Is something missing here?

Reply:
@jukkar Yes, for now only port A is used. B, E will be used when the switch support is added. C, D can be removed as there isn't any connections. I will remove it. Thanks.

Reply:
As @jukkar asked, lets remove PORTB & E for now.  Add them back when you have the additional code for the switch.